docs/_plugins/copy_api_dirs.rb

```diff
@@ -150,4 +150,31 @@
     cp("../R/pkg/DESCRIPTION", "api")
   end
 
+  if not (ENV['SKIP_SQLDOC'] == '1')
+    # Build SQL API docs
+
+    puts "Moving to project root and building API docs."
+    curr_dir = pwd
+    cd("..")
+
+    puts "Running 'build/mvn -DskipTests clean package' from " + pwd + "; this may take a few minutes..."
+    system("build/mvn -DskipTests clean package") || raise("SQL doc generation failed")
```

Comment:
Somehow, I can't run `./build/sbt clean package` and `spark-submit` in my environment -
 https://github.com/apache/spark/pull/17150#issuecomment-316950717

I tested this via `sbt` after manually reverting that commit in my environment. I will change this to `sbt` if anyone could confirm it was my mistake ...